src/pudl/metadata/fields.py

```diff
@@ -591,7 +773,11 @@
         "description": "Demand charges (USD).",
         "unit": "USD",
     },
-    "demand_mwh": {"type": "number", "unit": "MWh"},
+    "demand_mwh": {
+        "type": "number",
+        "description": "Demand in a given time frame.",
+        "unit": "MWh",
+    },
```

Comment:
This field comes from the `out_ferc714__hourly_predicted_state_demand `, maybe we should clarify in the column name that it's hourly demand? Or maybe the timestamps make it obvious.